Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -406,6 +406,7 @@
       - ConstructionBagOfHolding # Frontier
       - ServiceSelectiveDropper # Frontier
       - PlantAnalyzer # Frontier
+      - JanicartFlatpack # Frontier
```

Comment:
Could throw it in the service techfab for giggles.

Reply:
Agreed, gonna do it.